ruby_event_store-browser/lib/ruby_event_store/browser/app.rb

```diff
@@ -65,21 +65,13 @@ def self.for(event_store_locator:, host: nil, path: nil, api_url: nil, environme
             <head>
               <title>RubyEventStore::Browser</title>
               <link type="text/css" rel="stylesheet" href="<%= path %>/ruby_event_store_browser.css">
+              <meta name="ruby-event-store-browser-settings" content='<%= browser_settings %>'>
```

Comment:
`<meta>` surprised me a bit here, I'd have expected `data-*`:
https://html.spec.whatwg.org/multipage/dom.html#embedding-custom-non-visible-data-with-the-data-*-attributes

Reply:
Yeah, that's what I was planning on doing but then, when I started thinking about where to put this attribute (on what element) I realized that there's no "root" element here. So I went with how e.g. Rails includes the csrf token (https://api.rubyonrails.org/classes/ActionView/Helpers/CsrfHelper.html#method-i-csrf_meta_tags).

Seemed reasonable, but I can change it if you'd prefer to have it other way.